pyvenv.el

```diff
@@ -308,7 +302,7 @@ environment.
                  ;; https://github.com/jorgenschaefer/elpy/issues/144
                  (equal name nil)))
 
-    (unless (and use-existing (pyvenv--already-activated name))
+    (unless (equal pyvenv-virtual-env-name name)
```

Comment:
Oh, and I think this, together with the `when` above, is basically `(unless (member name (list "" nil pyvenv-virtual-env-name)) ...)`, no?

Reply:
I tried a few ways of combining all of that logic, but none of the alternatives expressed the idea (to me) as well as what I used. I'll be happy to change it, though, if you prefer.

Reply:
Just to make sure I haven't misunderstood: do you want me to make the change you mentioned?

Reply:
Yes … I think the two forms are equivalent, no?

Reply:
Yes, I agree that they're equivalent in terms of the logic. The more compact version is a bit more obscure to me, though that's just a matter of taste. I'll make the change and push it up.